tools/generate-record-row.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+"""
+Generate wheel metadata RECORD file row for given file.
+
+Usefull in cases where you want to postprocess the wheel archive e.g. include
+additional libraries after the wheel has been build.
+"""
+import argparse
+import base64
+import csv
+import hashlib
+import os
+import sys
+
+if sys.version_info[0] < 3:
```

Comment:
.. and then we can skip this block too. just require python3

Reply:
ack that will require bumping the version used to create wheels as well which we should also do. The question being should we do so here or in a follow up PR that bumps the interpreter version for our manylinux builds.

Reply:
We can skip it in this PR, but we'll need to address it soon since none of our py scripts work out of the box on when executed through shebang as `examples/eos-transactions.py` on ubuntu 20.04 (and soon others).

Reply:
agreed, cibuildwheel implies it will stop working with py2 as the execution environment in its release notes as well which will leave us with our hands tied. I'll get this in to 1.4.3 then work start attacking the migration